src/popup/components/Modals/AccountCreate.vue

```diff
@@ -6,46 +6,84 @@
     centered
     @close="resolve"
   >
-    <h1
```

Comment:
This is initially a `AccountCreateModal` component. It was renamed and one button added.